pkg/derr/errors.go

```diff
@@ -58,4 +58,5 @@ var (
 	ErrProtoValidate                 = cerr.NewAsertoError("E20050", codes.InvalidArgument, http.StatusBadRequest, "")
 	ErrGraphDirectionality           = cerr.NewAsertoError("E20052", codes.InvalidArgument, http.StatusPreconditionFailed, "unable to determine graph directionality")
 	ErrUnknownOpCode                 = cerr.NewAsertoError("E20053", codes.InvalidArgument, http.StatusPreconditionFailed, "unknown import OpCode value")
+	ErrMultipleIfMatchHeaders        = cerr.NewAsertoError("E20054", codes.InvalidArgument, http.StatusBadRequest, "to many If-Match headers, only one header of this type is accepted")
```

Comment:
I don't think we need an error for this case. If there are multiple if-match headers I'd just use the first one, and if we do want to return an error it can be ErrInvalidArgument with the message you have here.
I don't think this is an error that anyone would attempt to catch and handle.